### PR TITLE
Main test spec

### PR DESCRIPTION
**Pull request by SunnyAsar**

### CODE REVIEW

File: spec/main_test_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    end
+  end
+
+  describe '#my_none' do
+    it 'takes an array,checks than none match the rule' do
+      expect(array.none? { |i| i > 4 }).to eql(false)
+    end
+  end
+
+  context '#my_count' do
```
Comment (inhaq) [inline spec/main_test_spec.rb]: [Check](http://www.betterspecs.org/#contexts) the good practices of `context` and update accordingly

File: spec/main_test_spec.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# main tests
+require './bin/main.rb'
+RSpec.describe Enumerable do
+  array = [1, 2, 3, 4, 5]
```
Comment (inhaq) [inline spec/main_test_spec.rb]: Use `let` instead of local variable, how? [check it here](http://www.betterspecs.org/#let)

**Review by inhaq — CHANGES_REQUESTED** [review]
Hi, Good work. Your tests are running smooth but check out comments to follow best practices and will be helpful in the future. Kindly update and resubmit for review.

**Review by inhaq — APPROVED** [review]
Good job with learning `rspec` best practices. 👍🏻for the project. Go ahead and submit a project completion form.

**Comment by SunnyAsar** [comment]
thanks for the review @inhaq

**Pull request merged**